_posts/2020-06-05-ChrisDongB.md

```diff
@@ -0,0 +1,7 @@
+---
+title: getting out of this covid quarantine 
+author: chris
+date: 2020-06-05
+---
+
+I just want to go outside please
```

Comment:
We might not be able to go outside, but you win the award for best github username